src/ThemedStyleSheet.js

```diff
@@ -30,25 +34,28 @@ function resolve(...styles) {
   if (
     process.env.NODE_ENV !== 'production'
     && typeof performance !== 'undefined'
-    && performance.mark !== undefined
+    && performance.mark !== undefined && typeof performance.clearMarks === 'function'
   ) {
-    performance.mark('react-with-styles.resolve.start');
+    performance.clearMarks(START_MARK);
+    performance.mark(START_MARK);
   }
 
   const result = styleInterface.resolve(styles);
 
   if (
     process.env.NODE_ENV !== 'production'
     && typeof performance !== 'undefined'
-    && performance.mark !== undefined
+    && performance.mark !== undefined && typeof performance.clearMarks === 'function'
   ) {
-    performance.mark('react-with-styles.resolve.end');
+    performance.clearMarks(END_MARK);
+    performance.mark(END_MARK);
 
     performance.measure(
-      '\ud83d\udc69\u200d\ud83c\udfa8 [resolve]',
-      'react-with-styles.resolve.start',
-      'react-with-styles.resolve.end',
+      MEASURE_MARK,
+      START_MARK,
+      END_MARK,
     );
+    performance.clearMarks(MEASURE_MARK);
```

Comment:
doesn’t this mean that the performance measurement is immediately cleared? The point is to leave performance information in the dev tools for debugging.

Reply:
@ljharb it was the first thing that I checked.
So this is before changes:
![before](http://joxi.ru/a2XXaWliwMEP32.jpg)
And after:
![after](http://joxi.ru/Drl07zKTV5Y7Zr.jpg)